components/Tabs/Tabs.js

```diff
@@ -1,72 +1,85 @@
+    this.cards = Array.from(this.cards).map(element => new TabCard(element));
 
-     // Map over the newly converted NodeList we just created in our if statement above. Convert each this.cards element into a new instance of the TabCard class. Pass in a card object to the TabCard class. 
-    // this.cards = Array.from(this.cards).map();
+    // this.tabCard = new TabCard(this.cards[0]);
 
     // Add a click event that invokes this.selectTab
-    // this.tabElement.addEventListener();
-  }
+    this.tabElement.addEventListener('click', () => this.selectTab());
+  };
 
   selectTab(){
 
     // Select all elements with the .tab class on them
-    // const tabs = document.querySelectorAll();
+    const tabs = document.querySelectorAll('.tab');
     
     // Iterate through the NodeList removing the .active-tab class from each element
-    // tabs.forEach()
+    tabs.forEach(tabList => tabList.classList.remove('.active-tab'));
 
     // Select all of the elements with the .card class on them
-    // const cards = ;
-
+    const cards = document.querySelectorAll('.card');
+    // console.log(cards)
     // Iterate through the NodeList setting the display style each one to 'none'
-    // cards.forEach()
+    cards.forEach(tabList => tabList.style.display = "none");
     
     // Add a class of ".active-tab" to this.tabElement
-    // this.tabElement;
+    this.tabElement.classList.add('.active-tab');
   
-    // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class. Just un-comment the code and study what is happening here.
-    // this.cards.forEach(card => card.selectCard());
+    // Notice we are looping through the this.cards array and invoking selectCard() from the
+    // TabCard class. Just un-comment the code and study what is happening here.
+    this.cards.forEach(card => card.selectCard());
   }
-}
+};
 
 class TabCard {
   constructor(cardElement){
     // Assign this.cardElement to the cardElement DOM reference
-    // this.cardElement;
+    this.cardElement = cardElement;
   }
   selectCard(){
     // Update the style of this.cardElement to display = "flex"
-    // this.cardElement;
+    this.cardElement.style.display = "flex"
   }
 
-}
+};
 
 /* START HERE: 
 
 - Select all classes named ".tab" and assign that value to the tabs variable
 
-- With your selection in place, now chain a .forEach() method onto the tabs variable to iterate over the DOM NodeList
+- With your selection in place, now chain a .forEach() method onto the tabs variable
+to iterate over the DOM NodeList
 
-- In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
+- In your .forEach() method's callback function, return a new instance of TabLink
+and pass in each tab as a parameter
 
 */
-let tabs = document.querySelectorAll();
\ No newline at end of file
+let tabs = document.querySelectorAll('.tab');
+// console.log(tabs)
+tabs.forEach((tabs) => {
```

Comment:
I would call the parameter in your callback function tab, and not tabs.  each element is a single tab, not all the tabs.